practice_app/api/views.py

```diff
@@ -421,3 +440,73 @@ def user_registration(request):
     else:
         return JsonResponse({"status":"Username is already taken."}, status = 409)
 
+
+@csrf_exempt
+def post_papers(request):
+    # Authentication
+    if request.user.is_anonymous: # user is not logged in
+        if 'username' not in request.headers or 'password' not in request.headers: # credentials are incomplete
+            return JsonResponse({'status': 'username and password fields can not be empty'}, status=407)
+        username = request.headers['username']
+        password = request.headers['password']
+        user = authenticate(request, username=username, password=password)
+        if user == None: # credentials are incorrect
+            return JsonResponse({'status' : 'user credentials are incorrect.'},status=401)
+    else:
+        user = request.user
+
+    query = request.POST
+    if 'db' not in query.keys() or 'title' not in query.keys(): # db parameter or title parameter is not set
+        return JsonResponse({'status' : 'db and title parameters must be added to the request body.'},status=404)
```

Comment:
A more accurate status for this response would be 400, as the request is missing parameters and the syntax is malformed.